contracts/mod-nft/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 pub mod contract;
-pub mod error;
 pub mod msg;
-pub mod state;
+#[cfg(test)]
+mod tests;
+pub mod types;
```

Comment:
```suggestion
pub mod types;

pub use mod_sdk::error::ModError as ContractError;
```
Sorry for this :D Let's be consistent and call it as `ContractError` and make it public inside `lib.rs`